src/Microsoft.Performance.Toolkit.Engine.Tests/TestData/EngineBuildTabeTestDtos.cs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+
+namespace Microsoft.Performance.Toolkit.Engine.Tests.TestData
+{
+    [DataContract]
+    public class EngineBuildTableTestSuiteDto
```

Comment:
What is Dto?

Reply:
https://en.wikipedia.org/wiki/Data_transfer_object